---
## octomap_server.yaml

resolution: 0.05               # meters per voxel
frame_id: "odom"               # global frame for the map
base_frame_id: "taluy/base_link" # base frame of the robot
height_map: true
sensor_model:
  max_range: 5.0               # max sensor range to insert
  hit: 0.7                     # sensor model hit probability
  miss: 0.4                    # sensor model miss probability

# 2D projection settings
project_map: true              # enable /projected_map
publish_2d_map: true           # publish nav_msgs/OccupancyGrid
occupancy_min_z: -1.0
occupancy_max_z: 0.0
point_cloud_min_z: -100
point_cloud_max_z: -99              # ignore anything above 1.50 m (ceiling)
filter_ground: false           # try to remove the floor plane (disabled)

# optional: tune how often it repaints
publish_period: 1.0            # seconds between map publishes
